common/djangoapps/student/helpers.py

```diff
@@ -270,6 +261,28 @@ def get_next_url_for_login_page(request):
     return redirect_to
 
 
+def get_redirect_to(request):
+    """
+    Determine the redirect url and return if safe
+    :argument
+        request: request object
+
+    :returns: redirect url if safe else None
+    """
+    redirect_to = request.GET.get('next', None)
+
+    # if we get a redirect parameter, make sure it's safe. If it's not, drop the
+    # parameter.
+    if redirect_to and (not http.is_safe_url(redirect_to) or settings.STATIC_URL in redirect_to):
+        log.error(
```

Comment:
This should probably be a warning. An error could trigger alerts in New Relic and/or Splunk.